setup.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+from setuptools import setup
+
+
+def get_requirements(fn):
+    with open(fn, "r") as fp:
+        return [req.strip() for req in fp.readlines()]
```

Comment:
Assuming you have your `requrements.txt` file set up a certain way, then this reduces some duplication between this file and that one. If this gets in your way, you should stop doing this and just have two lists.

Reply:
Actually, this is a terrible idea because in `requirements.txt`, we pin to specific versions which is important for development. We don't want to do that in a package because it makes it muuuuch harder to use. I'll fix this.